internal/cloudapi/v2/v2_koji_test.go

```diff
@@ -392,8 +428,9 @@ func TestKojiCompose(t *testing.T) {
 			}`,
 		},
 	}
-	for _, c := range cases[2:3] {
-		test.TestRoute(t, handler, false, "POST", "/api/image-builder-composer/v2/compose", fmt.Sprintf(`
+	for idx, c := range cases {
+		t.Logf("Test case #%d", idx)
```

Comment:
`t.Run` is more idiomatic. (will make the review harder though so it should be done in a separate commit)

Reply:
Reworked based on your feedback.